codalab/lib/bundle_cli.py

```diff
@@ -1857,6 +1857,9 @@ def do_rm_command(self, args):
             '  search .limit=<limit>                  : Limit the number of results to the top <limit> (e.g., 50).',
             '  search .offset=<offset>                : Return results starting at <offset>.',
             '',
+            '  search .before=<datetime>              : Returns bundles created before given ISO 8601 timestamp (e.g. .before=2042-3-14 ). ',
```

Comment:
Do you want to add the `T00:00:00:00-08` to the end of this example so that it's consistent with the example timestamp given below?

Reply:
Both work so I wanted to provide two different examples.

Reply:
nit: `(e.g., .before=2042-3-14)` (note comma and spacing)

Reply:
Comment that both are inclusive